modules/couchdb/couchdb.go

```diff
@@ -0,0 +1,120 @@
+package couchdb
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+	"github.com/netdata/go.d.plugin/pkg/web"
+)
+
+func init() {
+	module.Register("couchdb", module.Creator{
+		Defaults: module.Defaults{
+			UpdateEvery: 10,
+		},
+		Create: func() module.Module { return New() },
+	})
+}
+
+func New() *CouchDB {
+	return &CouchDB{
+		Config: Config{
+			HTTP: web.HTTP{
+				Request: web.Request{
+					URL: "http://127.0.0.1:5984",
+				},
+				Client: web.Client{
+					Timeout: web.Duration{Duration: time.Second * 5},
+				},
+			},
+			Node: "_local",
+		},
+
+		urlPathOverviewStats: "/_node/%s/_stats",
+		urlPathSystemStats:   "/_node/%s/_system",
```

Comment:
i suggest to move these `urlPath*` to `collect.go` and make them constants so we have all used endpoints at the top of the `collect.go`

You still can use this fields and init them in Init (which doesn't really make sense for me, but it is ok)

Reply:
Moved them to the `collect.go`.